OpenRA.Mods.Common/Traits/Buildable.cs

```diff
@@ -34,6 +34,9 @@ public class BuildableInfo : TraitInfo<Buildable>
 		[Desc("Force a specific faction variant, overriding the faction of the producing actor.")]
 		public readonly string ForceFaction = null;
 
+		[Desc("Sequence of the actor that contains the cameo.")]
```

Comment:
The original games called the icons "cameo"s, but we don't except for here and the legacy sequence importer (where we can't really avoid it).  Can you please change "cameo" &rarr; "production icon" or "sidebar icon" here?